Modules/Camouflague.cs

```diff
@@ -53,7 +53,15 @@ public static void CheckCamouflage()
 
             if (oldIsCamouflage != IsCamouflage)
             {
-                Main.AllPlayerControls.Do(pc => Camouflage.RpcSetSkin(pc));
+                foreach (var pc in Main.AllPlayerControls)
+                {
+                    Camouflage.RpcSetSkin(pc);
```

Comment:
This has been this way since before the change, but `Camouflage` is redundant and should be removed on this occasion.

Reply:
Done